GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -92,20 +92,21 @@ protected override List<MapEntityMetadata> Parse()
                     {
                         XmlNode blockCoord = node.SelectSingleNode("property[@name='Position']");
                         string blockValue = blockCoord?.FirstChild.Attributes["value"].Value;
-
                         CoordS boundingBox = ParseCoord(blockValue ?? "0, 0, 0");
```

Comment:
This code seems duplicated in the if and else if blocks here, can move outside of if and else if blocks.